oml/datasets/list_.py

```diff
@@ -0,0 +1,105 @@
+from functools import lru_cache
+from pathlib import Path
+from typing import Dict, List, Optional, Sequence, Tuple, Union
+
+import albumentations as albu
+import numpy as np
+import torch
+from PIL import Image
+from torch.utils.data import Dataset
+
+from oml.exceptions import InvalidBBoxesException
+from oml.transforms.images.torchvision.transforms import get_normalisation_torch
+from oml.transforms.images.utils import TTransforms
+from oml.utils.images.images import TImReader, imread_cv2
+
+TBBox = Union[List[int], Tuple[int, int, int, int]]
```

Comment:
On one hand, I agree, on the other, It was @DaloroAT idea
https://github.com/OML-Team/open-metric-learning/pull/210#discussion_r1009367203

And I actually agree with it, considering how you define bboxes in this comment:
https://github.com/OML-Team/open-metric-learning/pull/210/files#r1014853658

Reply:
In that comment @DaloroAT most likely just tried to obtain the correct typing from the existing code rather than suggesting the idea of the type. I cannot ask him in person, because he will be on vacation the next days. So, let's keep the simple one `Tuple[int, int, int, int]`